exercises/practice/robot-simulator/test-robot-simulator.bats

```diff
@@ -0,0 +1,392 @@
+#!/usr/bin/env bats
+# generated on 
+load bats-extra
+
+assert_objects_equal() {
+    local result=$(
+        jq -n --argjson actual "$1" \
+              --argjson expected "$2" \
+            '$actual == $expected'
+    )
+    if [[ $result != "true" ]]; then
+        echo "expected: $2" >&2
+        echo "actual: $1" >&2
+        return 1
+    fi
+}
+
+@test 'Create robot:at origin facing north' {
+    #[[ $BATS_RUN_SKIPPED == "true" ]] || skip
+
+    run jq -c -f robot-simulator.jq << 'END_INPUT'
+      {
+        "robot": {
+          "position": {
+            "x": 0,
+            "y": 0
+          },
+          "direction": "north"
+        }
+      }
+END_INPUT
+
+    assert_success
+    expected='{"position":{"x":0,"y":0},"direction":"north"}'
```

Comment:
Is there any particular reason the expected doesn't have spaces after `:`?

Reply:
`jq -c` strips them all.